models.py

```diff
@@ -52,11 +54,24 @@ def __repr__(self):
         return '<id {}>'.format(self.code)
 
     def serialize(self):
+        links = [link.serialize() for link in self.links if link.active]
```

Comment:
Let me know if we want to keep inactive links in here!

Reply:
I think we do because it'll be good for the CMS user to see the links that they deactivated. These can be filtered out in the map frontend.